opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -101,7 +109,13 @@
 _server_active_requests_count_attrs_new = [
     HTTP_REQUEST_METHOD,
     URL_SCHEME,
-    # TODO: Support SERVER_ADDRESS AND SERVER_PORT
+]
+
+_server_active_requests_count_attrs_new_with_server_attributes = [
+    HTTP_REQUEST_METHOD,
```

Comment:
Nit: `_server_active_requests_count_attrs_new ` and append the two new attributes.